### ISSUE TITLE
Recent update to 1.51.1 triggers Windows Defender.

### ISSUE THREAD

**Issue by ent3r** [VSCodium/vscodium#564]
**Describe the bug**
After the recent update to 1.51.1, windows defender triggered on a VSCodium file.

**Please confirm that this problem is VSCodium-specific**
- [x] This bug doesn't happen if I use Microsoft's Visual Studio Code. It only happens in VSCodium.

**Please confirm that the issue/resolution isn't already documented**
- [x] I checked [the Docs page](https://github.com/VSCodium/vscodium/blob/master/DOCS.md) and my issue is not mentioned there.

**To Reproduce**
1. Be on version 1.51.0
2. Click on the auto update field
3. Auto update like normal
4. Start VSCodium
5. Windows Defender triggers

**Expected behavior**
Nothing happens, everything works as normal

**Screenshots**

![ApplicationFrameHost_09BANCR2rA](https://user-images.githubusercontent.com/32072697/99080530-b6c1c480-25c1-11eb-81ae-0a9b4e9e554c.png)

(Rough) translation:
> Warning level: Low
> Status: Active
> Date: 13th of November 2020, at 15:00
> Category: Potentially unwanted programs
> Details: This program has unwanted functions
> *Find out more*
> Affected elements:
>   file: *file path*

**Desktop (please complete the following information):**
 - OS: Windows 10 (10.0.19041)
 - Architecture: 64-Bit 
 - Version: 1.51.1

**Additional context**
*none*

**Comment by gbrdev** [comment]
I just had the same problem, Windows Defender quarantined my vscodium.
**I solved it by downloading the latest release again.**
https://github.com/VSCodium/vscodium/releases

**Comment by ent3r** [comment]
Defender didn't quarantine it for me, just warned me about that one file

**Comment by MadhanMurali** [comment]
Same issue. The setup file got removed (moved to quarantine) by windows defender as soon as the download finished. Downloaded from github.

**Comment by sdruskat** [comment]
Same here.

**Comment by paddle-to-the-sea** [comment]
First, thank you for all of your efforts on this project.

I have a similar issue.

Some days ago I downloaded VSCodiumSetup-x64-1.51.1.exe. I didn't get around to installing it, so it sat in my Downloads folder for a few days. (I'm sorry I don't have a more specific timeline than that.)

This morning, I was scanning some other files in my Downloads folder with Windows Defender, I got this "Threat found..." alert:
![Screenshot 2020-11-22 090443 - Copy](https://user-images.githubusercontent.com/46289457/99909112-33346180-2cac-11eb-9b78-718447a517c4.png)

It says that the threat detected is: Trojan:Win32/Wacatac.D3!ml and that the affected file is VSCodiumSetup-x64-1.51.1.exe.

Thanks again.
